src/snapshot.js

```diff
@@ -69,6 +69,31 @@ export function getPlayerSnapshot(player) {
   }
   snapshotObject.suppressedTracks = suppressedTracks;
 
+  /**
+   * iOS Safari will change caption mode to 'showing' if a user previously
+   * turned captions on manually for that video source, so this TextTrackList
+   * 'change' event handler will re-disable them in case that occurs during ad playback
+   */
+  const iOSTrackListChangeHandler = function() {
+    if (player.ads.isAdPlaying()) {
+      const textTrackList = player.textTracks();
+
+      for (let i = 0; i < textTrackList.length; i++) {
+        const track = textTrackList[i];
+
+        if (track.mode === 'showing') {
```

Comment:
I mean in `getPlayerSnapshot`, the snapshot object is built with i. a reference to the track, ii. the original track state. If Safari chooses to make a track's mode `showing` and the original track state saved in the snapshot isn't showing, should we update the original track state saved in the snapshot to be what the browser expects or leave it as it was when the snapshot was taken?

Let me know if that was confusing =P